gm4_ender_hoppers/data/gm4_ender_hoppers/loot_tables/destroy.json

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
We should probably follow the precedent of the other modules and call this `destroy_ender_hopper`